native/chat/inner-text-message.react.js

```diff
@@ -78,20 +71,11 @@ class InnerTextMessage extends React.PureComponent<Props> {
         onPress={this.props.onPress}
         onLongPress={this.props.onPress}
         activeOpacity={0.6}
+        style={[styles.message, messageStyle, cornerStyle]}
       >
-        <Hyperlink
-          linkDefault={true}
-          style={[styles.message, messageStyle, cornerStyle]}
-          linkStyle={linkStyle}
-        >
-          <Text
-            onPress={this.props.onPress}
-            onLongPress={this.props.onPress}
-            style={[textStyle, textCustomStyle]}
-          >
-            {text}
-          </Text>
-        </Hyperlink>
+        <Text style={[styles.text, textCustomStyle]}>
```

Comment:
I brought `styles.text` back here since this allows us to get rid of `state.linkPresent`. We need it before because the style inside the `text` rule was overriding the style from the `link` rule, but if we specify the `text` style at the root, this isn't an issue